src/ui/components/Timeline/index.tsx

```diff
@@ -380,8 +381,11 @@ class Timeline extends Component<PropsFromRedux> {
           </div>
           <Tooltip timelineWidth={this.overlayWidth} />
         </div>
-        <div className="timeline-time">
-          <span className="time-current">{getFormattedTime(currentTime)}</span>
+        <div
+          className="timeline-time text-right"
+          style={{ minWidth: `${formattedTime.length * 2 + 2}ch` }}
```

Comment:
Nice, i had no idea you could do this :P

Reply:
I also had forgotten about this unit of measure 😄